rpc/seele_rpc.go

```diff
@@ -213,20 +200,26 @@ func (rpc *SeeleRPC) GetBalance(address string) (int64, error) {
 	//   Balance:1.9975499e+12
 	//   Account:0x4c10f2cd2159bb432094e3be7e17904c2b4aeb21
 	// ]
-	account := result["Account"].(string)
-	if account != address {
-		return 0, fmt.Errorf("expected balance '%s', actually '%s'", address, result)
+
+	return getBalance(balanceMp, account)
+}
+
+// getBalance parse balance informations map to int64
+func getBalance(balanceMp map[string]interface{}, account string) (int64, error) {
+	retAccount := balanceMp["Account"].(string)
+	if retAccount != account {
+		return 0, fmt.Errorf("expected balance '%s', actually '%s'", account, retAccount)
 	}
-	balance := int64(result["Balance"].(float64))
-	return balance, nil
+	return int64(balanceMp["Balance"].(float64)), nil
```

Comment:
will data be lost converting float64 to int64?

Reply:
no, if a value is float64, I will not use int64 to receive it